ISHPullUp/ISHPullUpViewController.m

```diff
@@ -390,7 +401,10 @@ - (void)setBottomHeight:(CGFloat)bottomHeight animated:(BOOL)animated {
           initialSpringVelocity:config.initialVelocity
                         options:config.options
                      animations:updateBlock
-                     completion:nil];
+                     completion:^(BOOL finished) {
+                         self.isAnimatingStateChange = NO;
```

Comment:
Ok, it's probably better to falsely set it back a bit too early than to never set it back.